cobra/util/solver.py

```diff
@@ -386,3 +386,59 @@ def fix_objective_as_constraint(model, fraction=1):
         model.objective.expression,
         name=fix_objective_name, ub=ub, lb=lb)
     add_to_solver(model, constraint)
+
+
+def get_solution(model, reactions=None, metabolites=None):
```

Comment:
I figured I'm creating a solution from a solver state so it should go there. I have no preference, though, so I'll stick it in `solution.py`.